os_faults/common/service.py

```diff
@@ -127,6 +127,29 @@ def unplug(self, nodes=None):
                                      'action': 'block',
                                      'service': self.SERVICE_NAME}}, nodes)
 
+    @utils.require_variables('SERVICE_NAME')
+    def stresscpu(self, nodes=None):
+        nodes = nodes if nodes is not None else self.get_nodes()
```

Comment:
If it's not, the call fails, and that will propagate back up through ansible and rally.  Not sure it helps to discover it earlier in this case.